src/components/footer/footer.jsx

```diff
@@ -0,0 +1,56 @@
+// Bootstrap CSS
+import "bootstrap/dist/css/bootstrap.min.css";
+// Bootstrap Bundle JS
+import "bootstrap/dist/js/bootstrap.bundle.min";
+
+import logo from './2.png';
+
+function Footer() {
+    return(
+        <footer>
+    <div class="container-fluid bg-black text-light p-4">
+        <div class="row row-cols-4 ">
```

Comment:
le code n’est pas bien formatter. Le `div` doit être imbriqué au `footer`. Cela va rendre ça plus lisible et propre. Pour cela il faut `npm install --save-dev eslint prettier` et plus d’explications [https://javascript.plainenglish.io/setting-eslint-and-prettier-on-a-react-typescript-project-2021-22993565edf9](ici)